sormas-ui/src/main/java/de/symeda/sormas/ui/samples/PathogenTestController.java

```diff
@@ -151,7 +194,15 @@ public CommitDiscardWrapperComponent<PathogenTestForm> getPathogenTestEditCompon
 		if (isEditOrDeleteAllowed) {
 			editView.addCommitListener(() -> {
 				if (!form.getFieldGroup().isModified()) {
-					savePathogenTest(form.getValue(), false);
+					PathogenTestDto editedPathogenTest = form.getValue();
+					if (editedPathogenTest.getSample() != null) {
+						savePathogenTest(form.getValue(), false);
```

Comment:
please rename `savePathogenTest` tobe similar to the one for env samples

Reply:
renamed